packages/editor-ui/src/mixins/workflowHelpers.ts

```diff
@@ -747,6 +747,22 @@ export const workflowHelpers = mixins(
 				} catch (error) {
 					this.uiStore.removeActiveAction('workflowSaving');
 
+					if (error.errorCode === 400 && error.message.startsWith('Your most recent changes may be lost')) {
```

Comment:
I think linking the BE text to the error might be a problem in the future. We either need to write tests to assert this behavior so it doesn't break or simply make sure that the http status code can only mean this.

One possibility is to add an error code as part of the output so you can use this WDYT?

Reply:
I actually wanted to discuss this with you. Let's use a custom error code/error id to detect when this occurs.